root/edit/details/AddSeries.js

```diff
@@ -0,0 +1,71 @@
+/*
+ * @flow
+ * Copyright (C) 2020 Anirudh Jain
+ *
+ * This file is part of MusicBrainz, the open internet music database,
+ * and is licensed under the GPL version 2, or (at your option) any
+ * later version: http://www.gnu.org/licenses/gpl-2.0.txt
+ */
+
+import React from 'react';
+
+import {addColonText} from '../../static/scripts/common/i18n/addColon';
+import EntityLink from '../../static/scripts/common/components/EntityLink';
+
+type AddSeriesEditProps = {
+  ...EditT,
+  +display_data: {
+    +comment: string,
+    +name: string,
+    +ordering_type: SeriesOrderingTypeT,
+    +series: SeriesT,
+    +type: SeriesTypeT,
+  },
+};
+
+type Props = {
+  edit: AddSeriesEditProps,
+};
+
+const AddSeries = ({edit}: Props) => {
+  return (
+    <>
+      <table className="details">
+        <tbody>
+          <tr>
+            <th>{addColonText(l('Series'))}</th>
+            <td><EntityLink entity={edit.display_data.series} /></td>
+          </tr>
+        </tbody>
+      </table>
+      <table className="details add-series">
+        <tbody>
+          <tr>
+            <th>{addColonText(l('Name'))}</th>
+            <td>{edit.display_data.name}</td>
+          </tr>
+          {edit.display_data.comment ? (
+            <tr>
+              <th>{addColonText(l('Disambiguation'))}</th>
+              <td>{edit.display_data.comment}</td>
+            </tr>
+          ) : null}
+          {edit.display_data.type ? (
+            <tr>
+              <th>{addColonText(l('Type'))}</th>
+              <td>{edit.display_data.type.name}</td>
+            </tr>
+          ) : null}
+          {edit.display_data.ordering_type ? (
+            <tr>
+              <th>{addColonText(l('Ordering Type'))}</th>
+              <td>{edit.display_data.ordering_type.name}</td>
```

Comment:
I think this should be translatable too, but I'm not sure how exactly. @mwiencek, would this be l_attributes with ordering_type or something? (there's l_name but we shouldn't use that).

Reply:
I think:

```suggestion
              <td>
                {lp_attributes(
                  edit.display_data.ordering_type.name,
                  'series_ordering_type',
                )}
              </td>
```

Reply:
thanks :)